apps/frontend-v3/app/(marketing)/risks/page.tsx

```diff
@@ -982,8 +983,22 @@ export default function Privacy() {
                     <p>
                       Oracles are data providers which supply external information to smart
                       contracts. Oracles, like Chainlink, may be used to source exchange rates
-                      between pool tokens for a rate provider in Balancer MetaStable pools. The
-                      risks of using Oracles to supply exchange rates include:
+                      between pool tokens for a rate provider in Balancer MetaStable pools.
+                    </p>
+                    <p>
+                      Arbitrageurs can exploit oracle latency by frontrunning oracle updates at 
+                      direct expense to liquidity providers. If an arbitrageur sees an oracle update 
+                      in the mempool that will significantly change an asset's price, they can 
+                      anticipate the resulting market movements by buying/selling the asset before 
+                      the update takes effect. Given there could be a slight delay between price feeds,
+                      there’s a potential window for arbitrage.
+                    </p>
+                      In a similar risk, blockchain validators could censor updates, by choosing to 
```

Comment:
There is a missing opening `<p>` tag here that destroys the rest of the page format and probably makes some of the checks fail.